core/Command/User/Setting.php

```diff
@@ -213,13 +213,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 				return 1;
 			}
 		} else {
-			$settings = $this->getUserSettings($uid, $app);
+			$ignoreMissingUser = $input->getOption('ignore-missing-user');
+			$settings = $this->getUserSettings($uid, $app, $ignoreMissingUser);
 			$this->writeArrayInOutputFormat($input, $output, $settings);
 			return 0;
 		}
 	}
 
-	protected function getUserSettings($uid, $app) {
+	protected function getUserSettings(string $uid, string $app, bool $ignoreMissingUser = false) {
```

Comment:
```suggestion
	protected function getUserSettings(string $uid, string $app): array {
```

The bool is unused now.